app/views/games/show.html.erb

```diff
@@ -1,5 +1,9 @@
 <p id="notice"><%= notice %></p>
 
+<div class="white-player-image">
+  <%= image_tag @game.white_player.gravatar_url %>
+</div>
```

Comment:
Looks good!, then for black_player, can do the `present?` check like talked in Slack.

😄